### PR TITLE
Run black on enable

### PR DESCRIPTION
**Pull request by jwiggins**
Step one towards running `flake8` on the code.

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
LGTM. I obviously didn't go through the entire diff but i'm reasonably confident that black will do the right thing. @jwiggins did you have to manually edit/modify/undo anything black did to the codebase?

**Comment by jwiggins** [comment]
> did you have to manually edit/modify/undo anything black did to the codebase?

Yes

**Comment by rahulporuri** [comment]
Still LGTM

**Review by rahulporuri — APPROVED** [review]
I went through this PR with a very _very_ fine toothed comb and apart from the following suggestions/comments, this PR still LGTM.

Apart from that, the whole process made me feel reviewing the output of black with a fine toothed comb on a large package like enable is IMO not worth the return on the time investment.

File: enable/wx/scrollbar.py [hunk 1]
```diff
@@ -24,11 +23,19 @@ def valid_range(object, name, value):
...
             line_size = max(min(line_size, page_size), 0.0)
-            return (float(low), float(high), float(page_size), float(line_size))
-    except:
+            return (
+                float(low),
+                float(high),
+                float(page_size),
+                float(line_size),
+            )
+    except Exception:
```
[the remark this answers is not in the document]
Comment (jwiggins) [reply enable/wx/scrollbar.py:32]: Nope. That was my intervention.

File: enable/wx/cairo.py [hunk 2]
```diff
@@ -36,12 +36,17 @@ def _window_paint(self, event):
 
         control = self.control
         pixel_map = self._gc.pixel_map
-        wdc = control._dc = wx.PaintDC(control)
```
[the remark this answers is not in the document]
Comment (jwiggins) [reply enable/wx/cairo.py:39]: Again, human touch

File: enable/tools/viewport_pan_tool.py [hunk 3]
```diff
@@ -87,8 +88,8 @@ def dragging(self, event):
             new_position[ndx] -= delta
 
         if self.constrain:
-            self.component.view_position[self.constrain_direction] = \
-                        new_position[self.constrain_direction]
+            _dir = self.constrain_direction
+            self.component.view_position[_dir] = new_position[_dir]
```
[the remark this answers is not in the document]
Comment (jwiggins) [reply enable/tools/viewport_pan_tool.py:92]: Manual. Black is dumb.

File: enable/savage/svg/pathdata.py [hunk 4]
```diff
@@ -27,102 +28,118 @@ class CaselessPreservingLiteral(CaselessLiteral):
...
 
-closePath = Group(Command("Z")).setParseAction(lambda t: ('Z', (None,)))
+closePath = Group(Command("Z")).setParseAction(lambda t: ("Z", (None,)))
 
 flag = oneOf("1 0").setParseAction(lambda t: bool(int((t[0]))))
 
 arcRadius = (
-    nonnegativeNumber + maybeComma + #rx
-    nonnegativeNumber #ry
+    nonnegativeNumber + maybeComma + nonnegativeNumber  # rx  # ry
```
[the remark this answers is not in the document]
Comment (jwiggins) [reply enable/savage/svg/pathdata.py:117]: What a strange edit from black. This one wasn't me.

File: enable/examples/_etsdemo_info.py [hunk 5]
```diff
@@ -32,7 +32,5 @@ def info(request):
     return {
         "version": 1,
         "name": "Enable Examples",
-        "root": (
-            pkg_resources.resource_filename("enable.examples", "demo")
-        )
+        "root": (pkg_resources.resource_filename("enable.examples", "demo")),
```
[the remark this answers is not in the document]
Comment (jwiggins) [reply enable/examples/_etsdemo_info.py:35]: Yeah. I caught this in `kiva` too

**Comment by jwiggins** [comment]
Thanks for the review!

**Pull request merged**